pkg/controller/storagecluster/cephcluster.go

```diff
@@ -370,37 +406,51 @@ func newStorageClassDeviceSets(sc *ocsv1.StorageCluster) []rook.StorageClassDevi
 			prepareplacement := rook.Placement{}
 
 			if noPlacement {
-				in := getPlacement(sc, "osd")
-				(&in).DeepCopyInto(&placement)
+				if supportTSC {
+					in := getPlacement(sc, "osd-tsc")
+					(&in).DeepCopyInto(&placement)
+				} else {
+					in := getPlacement(sc, "osd")
+					(&in).DeepCopyInto(&placement)
 
-				if len(topologyKeyValues) >= replica {
-					placement.PodAntiAffinity.PreferredDuringSchedulingIgnoredDuringExecution[0].PodAffinityTerm.TopologyKey = topologyKey
+					if len(topologyKeyValues) >= replica {
+						placement.PodAntiAffinity.PreferredDuringSchedulingIgnoredDuringExecution[0].PodAffinityTerm.TopologyKey = topologyKey
```

Comment:
This commit shows the white space difference in UI, unable to catch this in IDE as well as in git diff. not sure why its showing in here.